openapi/godaddy.subscriptions/client.bal

```diff
@@ -15,33 +15,31 @@
 // under the License.
 
 import ballerina/http;
-import ballerina/url;
-import ballerina/lang.'string;
 
 # Provides API key configurations needed when communicating with a remote HTTP endpoint.
-@display {label: "GoDaddy Subscriptions", iconPath: "icon.png"}
 public type ApiKeysConfig record {|
-    # API keys related to connector authentication
-    map<string> apiKeys;
+    string authorization;
 |};
 
 # This is a generated connector for [GoDaddy Subscriptions API v1](https://developer.godaddy.com/doc/endpoint/subscriptions) OpenAPI specification.
-# The GoDaddy subscriptions API provides capability to access GoDaddy operations related to subscriptions.
+# The GoDaddy Subscriptions API provides capability to access GoDaddy operations related to subscriptions.
+@display {label: "GoDaddy Subscriptions", iconPath: "icon.png"}
 public isolated client class Client {
     final http:Client clientEp;
-    final readonly & map<string> apiKeys;
+    final readonly & ApiKeysConfig apiKeyConfig;
     # Gets invoked to initialize the `connector`.
     # The connector initialization requires setting the API credentials. 
     # Create a [GoDaddy](https://sg.godaddy.com/) and obtain tokens by following [this guide](https://developer.godaddy.com/getstarted).
     #
-    # + apiKeyConfig - Provide your API Key as `Authorization`. Eg: {Authorization : sso-key [<API_KEY>]:[<API_SECRET>]} 
+    # + apiKeyConfig - API keys for authorization 
     # + clientConfig - The configurations to be used when initializing the `connector` 
     # + serviceUrl - URL of the target service 
     # + return - An error if connector initialization failed 
     public isolated function init(ApiKeysConfig apiKeyConfig, http:ClientConfiguration clientConfig =  {}, string serviceUrl = "//api.ote-godaddy.com/") returns error? {
```

Comment:
Invalid default service URL is used here in the method signature. Please fix if possible.

Reply:
Fixed in https://github.com/ballerina-platform/openapi-connectors/commit/e2ec5928799e05c94828171641d94ac96370565e